.env.development

```diff
@@ -0,0 +1,3 @@
+ENVIRONMENT = development
```

Comment:
We don't need dev and production files, these files may contain sensitive data. You can add .env.example with empty values